lib/cucumber/blendle/version.rb

```diff
@@ -6,6 +6,6 @@ module Cucumber
   # This module defines default steps used by all of Blendle Ruby projects.
   #
   module BlendleSteps
-    VERSION = '0.9.0'.freeze
+    VERSION = '0.9.1'.freeze
```

Comment:
@JeanMertz once told me it's bad practice to update version numbers in the same PR as a new feature. Not sure if I've remembered this correctly.